aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -81,6 +81,7 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.type_info.type_name.base, "type_info.type_name.base", 300 * MUL],
     // TODO(Gas): the on-chain name is wrong...
     [.type_info.type_name.per_byte_in_str, "type_info.type_name.per_abstract_memory_unit", 5 * MUL],
+    [.type_info.chain_id.base, optional "type_info.chain_id.base", 150 * MUL],
```

Comment:
How did you come up with 150?

Reply:
It's the minimum gas cost. 

The fetching of the chain ID from storage (when the VM is created) will be very fast due to caching, so I decided to make this cheap.